packages/compat/src/resolver-transform.ts

```diff
@@ -323,15 +320,15 @@ class TemplateResolver implements ASTPlugin {
   }
 
   private get staticComponentsEnabled(): boolean {
```

Comment:
All three of these private getters can get replaced with a single staticInvokablesEnabled